src/main/java/org/commonjava/migrate/pathmap/CassandraMigrator.java

```diff
@@ -46,10 +57,32 @@
 
     private final ChecksumCalculator checksumCalculator;
 
+    private final PreparedStatement preparedStoresIncrement;
+
+    private final String keyspace = "indycache";
```

Comment:
Not sure if this keyspace is same with path-mapped-storage one. If same, I think this keyspace is passed from command options. See MigrateOptions.getCassandraKeyspace